### PR TITLE
changed plot to draw-plot and tweaked prose accordingly

### PR DESCRIPTION
**Pull request by taps3**

### CODE REVIEW

**Comment by sorawee** [comment]
Did you change the function name?

**Comment by schanzer** [comment]
Added a function (rather than method) version of the `series.quickdisplay()` idea you and @jpolitz mentioned earlier.

File: courses/data-science/units/unit1/the-unit.scrbl [hunk 1]
```diff
@@ -661,14 +661,12 @@
                         We can define identifiers for both the series and the plot:
                         @code[#:multi-line #t]{
                             # define the series and the graph for the function f(x)=√x
-                            sqrt-series = function-plot(num-sqrt)
-                            sqrt-graph  = plot(sqrt-series)
+                            sqrt-plot = function-plot(num-sqrt)
+                            sqrt-graph  = draw-plot(sqrt-plot)
```
Comment (sorawee) [inline courses/data-science/units/unit1/the-unit.scrbl]: According to the contract above, `draw-plot` consumes a string (title) and then a series. Here it has only one argument

Comment (sorawee) [inline courses/data-science/units/unit1/the-unit.scrbl]: Oh shoot, this was merged already... @schanzer

**Pull request merged**